### PR TITLE
Fixes #27356 - Remove global vars from eslint

### PR DESCRIPTION
**Pull request by amirfefer**
Now we can import translation functions directly from `i18n` module, no need to use global vars for it nor for jquery.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -16,10 +16,6 @@
     "navigator": false,
     "unescape": false,
     "window": false,
-    "$": true,
-    "_": true,
-    "__": true,
-    "n__": true,
     "d3": true
```
Comment (ezr-ondrej) [inline .eslintrc]: do we need to keep d3?

Comment (sharvit) [reply .eslintrc]: +1 for removing d3.
We can put comments where we do use it.

Actually, can't we remove the whole `globals` field?

Comment (tbrisker) [reply .eslintrc]: ping @amirfefer - any updates on this?

**Comment by amirfefer** [comment]
@ezr-ondrej  took some time but d3 is out from global vars :)

**Comment by tbrisker** [comment]
@amirfefer could you rebase on latest develop please to make sure tests still pass?

File: .eslintrc [hunk 2]
```diff
@@ -16,10 +16,5 @@
     "navigator": false,
     "unescape": false,
     "window": false,
```
Comment (sharvit) [inline .eslintrc]: IMO we should remove all globals as I don't see a reason continue using it.

**Review by sharvit — COMMENTED** [review]
@amirfefer can we remove all globals?

**Comment by amirfefer** [comment]
Thanks @sharvit , @tbrisker 
Rebased + all the global are removed

**Review by tbrisker — APPROVED** [review]
Thanks @amirfefer !

**Pull request merged**